src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Lzcnt.cs

```diff
@@ -0,0 +1,27 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// Lzcnt.cs
```

Comment:
Shall I use XML docs for each function's comment, e.g., `// unsigned int _lzcnt_u32 (unsigned int a)` ?